cmd/humioctl/filter_alerts_list.go

```diff
@@ -0,0 +1,58 @@
+// Copyright © 2020 Humio Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"strings"
+
+	"github.com/humio/cli/cmd/internal/format"
+	"github.com/spf13/cobra"
+)
+
+func newFilterAlertsListCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "list [flags] <view>",
+		Short: "List all filter alerts in a view.",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			view := args[0]
+			client := NewApiClient(cmd)
+
+			filterAlerts, err := client.FilterAlerts().List(view)
+			exitOnError(cmd, err, "Error fetching filter alerts")
+
+			var rows = make([][]format.Value, len(filterAlerts))
+			for i := range filterAlerts {
+				filterAlert := filterAlerts[i]
+
+				var actionNames = make([]string, len(filterAlert.Actions))
+				for j := range filterAlert.Actions {
+					actionNames[j] = filterAlert.Actions[j]
+				}
+
+				rows[i] = []format.Value{
+					format.String(filterAlert.Name),
+					format.Bool(filterAlert.Enabled),
+					format.String(filterAlert.Description),
+					format.String(strings.Join(actionNames, ", ")),
+				}
+			}
+
+			printOverviewTable(cmd, []string{"Name", "Enabled", "Description", "Actions"}, rows)
```

Comment:
Why only these columns?

Reply:
I guess it's the same question to as why we emit the warnings/last error...

What are the needs of the cli user? Which variables are interesting?

Reply:
```
        NAME        | ENABLED |    DESCRIPTION     | ACTIONS |    LABELS    |      RUN AS USER ID      | QUERY OWNERSHIP TYPE  
+-------------------+---------+--------------------+---------+--------------+--------------------------+----------------------+
  some-filter-alert | false   | some-description 2 | testing | some-label-2 | xxx | User    
```